# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/res/prim_shared.glsl
# Commit: 0c03e3ad5e56
# Full Hash: 0c03e3ad5e56201e35437734b0ae1d34faa52005
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2020-12-15 09:29:54
# Regressor Bug: 1682365
# File Overlap Count: 1
# Description:
#   Bug 1682365 - Pt 1 - Remove PrevPassAlpha and PrevPassColor r=nical
#   
#   Remove usage of the implicit prev pass alpha and color texture
#   samplers from batching / renderer / shader code. They are replaced
#   by explicit references to the texture ID for the source task.
# ==============================================================================

diff -r 4e0ba157c016 -r 0c03e3ad5e56 gfx/wr/webrender/res/prim_shared.glsl
--- a/gfx/wr/webrender/res/prim_shared.glsl	Sat Dec 12 22:13:43 2020 +0000
+++ b/gfx/wr/webrender/res/prim_shared.glsl	Mon Dec 14 19:38:48 2020 +0000
@@ -15,8 +15,6 @@
 #define RASTER_LOCAL            0
 #define RASTER_SCREEN           1
 
-uniform sampler2DArray sPrevPassAlpha;
-uniform sampler2DArray sPrevPassColor;
 uniform sampler2DArray sClipMask;
 
 vec2 clamp_rect(vec2 pt, RectWithSize rect) {